SimplePayment.Test/UnitTest1.cs

```diff
@@ -7,12 +12,67 @@ public class Tests
         [SetUp]
         public void Setup()
         {
+            TestBillingDetailModel();
+            TestPaymentResponseModel();
+        }
+        public string RemoveWhiteSpace(string json)
+        {
+            return Regex.Replace(json, @"(""[^""\\]*(?:\\.[^""\\]*)*"")|\s+", "$1");
         }
 
         [Test]
-        public void Test1()
+        public void TestBillingDetailModel()
         {
-            Assert.Pass();
+            var billingDetailsJson = @"{
+                ""Name"" : ""Teszt B�la"", 
+                ""Company"" : ""Teszt Kft."", 
+                ""Country"" : ""Magyarorsz�g"", 
+                ""City"" : ""Szeged"", 
+                ""State"" : ""Csongr�d"", 
+                ""Zip"" : ""6722"", 
+                ""Address"" : ""Teszt utca 7."", 
+                ""Address2"" : ""2 / 3"", 
+                ""Phone"":""36701234567""
+            }";
```

Comment:
Az ilyen jellegű szöveget, ami gyakorlatilag egy json, én kiraknám külön egy json file-ba a test projectbe, lehetne neki egy mappa, hogy TesJson pl., és akkor onnan beolvasni. De egyébként így is jól van. Kicsit olvashatóság miatt is megint, kevesebb lenne az ilyen statikus adat itt, és a test könnyebben átlátható talán. És ehhez pl. lehetne egy helper method, ami kiolvassa file-ból a dolgokat.
De ez szintén kicsit személyes vélemény :)